arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Iterable.kt

```diff
@@ -330,57 +330,6 @@ public fun <A> Iterable<Result<A>>.sequenceResult(): Result<List<A>> =
 public fun <A> Iterable<Result<A>>.sequence(): Result<List<A>> =
   traverse(::identity)
 
-@Deprecated("traverseValidated is being renamed to traverse to simplify the Arrow API", ReplaceWith("traverse(semigroup, f)", "arrow.core.traverse"))
-public inline fun <E, A, B> Iterable<A>.traverseValidated(
-  semigroup: Semigroup<E>,
-  f: (A) -> Validated<E, B>
-): Validated<E, List<B>> =
-  traverse(semigroup, f)
-
-@OptIn(ExperimentalTypeInference::class)
-@OverloadResolutionByLambdaReturnType
-public inline fun <E, A, B> Iterable<A>.traverse(
```

Comment:
Replacing with `mapAccumulating` is for a follow-up PR?

Reply:
yes

Reply:
what do you mean with the issue with the git history? Am I doing something wrong? how should I do it?